core/arch/arm/plat-stm32mp1/link.mk

```diff
@@ -0,0 +1,25 @@
+include core/arch/arm/kernel/link.mk
+
+# Create stm32 formatted images from the native binary images
+
+define stm32image_cmd
+	@$(cmd-echo-silent) '  GEN     $@'
+	$(q)./core/arch/arm/plat-stm32mp1/scripts/stm32image.py \
+		--load 0 --entry 0
+endef
+
+all: $(link-out-dir)/tee-header_v2.stm32
+cleanfiles += $(link-out-dir)/tee-header_v2.stm32
+$(link-out-dir)/tee-header_v2.stm32: $(link-out-dir)/tee-header_v2.bin
+	$(stm32image_cmd) --source $< --dest $@
+
+all: $(link-out-dir)/tee-pager_v2.stm32
+cleanfiles += $(link-out-dir)/tee-pager_v2.stm32
+$(link-out-dir)/tee-pager_v2.stm32: $(link-out-dir)/tee-pager_v2.bin
+	$(stm32image_cmd) --source $< --dest $@
+
+all: $(link-out-dir)/tee-pageable_v2.stm32
+cleanfiles += $(link-out-dir)/tee-pageable_v2.stm32
+$(link-out-dir)/tee-pageable_v2.stm32: $(link-out-dir)/tee-pageable_v2.bin
+	@[ `stat -c %s $<` -ne 0 ] || echo "" > $<
```

Comment:
Looks like a hack ;-) A rule should not modify its prerequisites. How about fixing `stm32image.py` so that it deals with empty files instead?

Reply:
:) ok